gh_pages/_source/session3/Build-a-Moveit!-Package.md

```diff
@@ -41,7 +41,7 @@ In this exercise, you will generate a MoveIt package for the UR5 workcell you bu
        type = 'fixed'
        ```
 
-    1. Add a planning group called `manipulator` that names the kinematic chain between `base_link` and `tool0`.
+    1. Add a planning group called `manipulator` that names the kinematic chain between `base_link` and `tool0`. Note: Make sure there are no spaces after the planning group name. 
```

Comment:
I'd rephrase this as:
```
Make sure there are no space characters before or after the group name
```

Reply:
Perhaps even more generic: follow ROS naming guidelines/requirements and don't use *any* whitespace, anywhere.

Reply:
@gavanderhoorn @jrgnicho  Thanks for the comments. I have made the required changes.